### PR TITLE
Relabel stack to node red version

### LINKED ISSUE #3914: Relabelling "Stack" to "Node-RED Version"
Our own use case of "Stacks" has been entirely bound to a Node-RED Version. As a new user coming in "Stack" is unlikely to be clear as a term - are we not better off just calling it "Node-RED Version"?

### PR DESCRIPTION
**Pull request by cstns**
## Description

Updated ui to reflect new nomenclature for stacks for non-admin users.

## Related Issue(s)

closes #3914
part of #3884

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by cstns** [comment]
Not being a native English speaker some of the finer touches might have eluded me, so an additional check would be more than welcomed!

With the change in nomenclature some of the existing messages would also need touch-ups. Ex:
```
                        There is a new version of the current Node-RED Version available.
                        Updating the Node-RED Version will restart the instance.
```

**Comment by cstns** [comment]
Updated to
```
There is a new version of Node-RED available.
Updating the Node-RED Version will restart the instance.
```
As per @joepavitt 's recommendation

**Review by joepavitt — COMMENTED** [review]
Not sure about some code included, have queries - all good otherwise

File: frontend/src/pages/team/createApplication.vue [hunk 1]
```diff
@@ -18,25 +12,28 @@
...
+            :applications="applications"
+            :applicationSelection="applicationCreated"
+            :team="team"
+            :applicationFieldsLocked="applicationCreated"
+            :applicationFieldsVisible="true"
+            :billing-enabled="!!features.billing"
+            :flow-blueprints-enabled="!!features.flowBlueprints"
+            :submit-errors="errors"
+            @on-submit="handleFormSubmit"
+        />
```
Comment (joepavitt) [inline frontend/src/pages/team/createApplication.vue:36]: Is this meant to be in here? Looks related to another PR for refreshing the Applications view?

Comment (cstns) [reply frontend/src/pages/team/createApplication.vue:36]: Must have sliped in from another task I was doing in parallel. Reverting

**Review by joepavitt — APPROVED** [review]

**Pull request merged**